docs/policy/rdt.md

```diff
@@ -10,43 +10,116 @@ control groups. Resource allocation is specified on the group level and each
+contrast to partitions, cache allocation between classes under a specific
+partition may overlap (and they usually do).
 
-CRI-RM has a built-in default configuration containing three classes
-corresponding to the Pod QOS classes of Kubernetes. These are utilized by the
-`static` policy.
+The set of RDT classes can be freely specified, but, it should be ensured that
+classes corresponding the Pod QoS classes are specified. Also, the maximum
+number of classes (CLOSes) supported by the underlying most not be exceeded.
+
+### Example
+
+Below is a config snippet that would allocate (ca.) 60% of the cache lines
+exclusively to the Guarenteed class. The remaining 40% is for Burstable and
+Besteffort, Besteffort getting only 50% of this.
+
+```
+metadata:
+  name: cri-resmgr-config.default
+  namespace: kube-system
+data:
+...
+  rdt: |+
+    # Common options
+    options:
+      l3:
+        # Make this false if CAT must be available
+        optional: true
+    partitions:
+      exclusive:
+        l3Allocation:
+          # Allocate 60% of all cache IDs to the "exclusive" partition
+          all: "60%"
+        classes:
+          Guaranteed:
+            l3schema:
+              # Allocate all of the partitions cache lines to "Guarenteed"
+              all: "100%"
+      shared:
+        l3Allocation:
+          # Allocate 40% of all cache IDs to the "shared" partition
+          # These will NOT overlap with the cache lines allocated for "exclusive" partition
+          all: "40%"
+        classes:
+          Burstable:
+            l3schema:
+              # Allow "Burstable" to use all cache lines of the "shared" partition
+              all: "100%"
+          BestEffort:
+            l3schema:
+              # Allow "Besteffort" to use half of the cache lines of the "shared" partition
+              # These will overlap with those used by "Burstable"
+              all: "50%"
+```
+
+The configuration also supports far more fine-grained control, e.g. per
+cache-ID configuration (i.e. different sockets having different allocation) and
+Code and Data Prioritization (CDP) allowing different allocation for code and
+data paths.
+
+In addition, if the hardware details are known, raw bitmasks or bit numbers
+("0x1f" or 0-4) can used instead of percentages in order to be able to
+configure allocations exactly as required. The bits in this case are
+corresponding those in /sys/fs/resctrl/ bitmasks. You can also mix relative
+(percentage) and absolute (bitmask) allocations.
 
 See `rdt` in the [example ConfigMap spec](/sample-configs/cri-resmgr-configmap.example.yaml)
-for an example configuration.
+for a more complete example configuration.
+
+### Dynamic Configuration
+
+RDT supports dynamic configuration i.e. the resctrl filesystem is reconfigured
+whenever a configuration update e,g, via the [Node Agent](../node-agent.md) is
+received. However, the configuration update is rejected if is incompatible with
```

Comment:
typo: ...rejected if it is incompatible with

Reply:
fixed